aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +252,41 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form guidance
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
+- Use a single-column full-page form instead of multi-column layouts.
+- Full-page forms shoud be center-aligned.
+- Use full-page when the form is too long.
+- Use Center modal instead of side drawers/layers for secondary inputs.
+- Use side layers to display help text, filters, or a form over another form.
```

Comment:
This seems to contradict the bullet point above that says not to use a side drawer for a form over a form.

Reply:
Ill check on this